packages/cactus-plugin-ledger-connector-fabric/src/main/json/openapi.json

```diff
@@ -286,6 +286,42 @@
                         "type": "string"
                     }
                 }
+            },
+            "PrometheusExporterOptions": {
```

Comment:
@jagpreetsinghsasan Do we still need this in the request? IIRC HTTP GET requests cannot have a body.

Reply:
Fixed